### PR TITLE
HPCC-30177 Bump vcpkg versions for 9.4.x

### PR DESCRIPTION
**Pull request by GordonSmith**
## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [x] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [ ] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [x] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [ ] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Cloud-compatibility
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [x] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

### CODE REVIEW

File: dockerfiles/platform-build/Dockerfile [hunk 1]
```diff
@@ -76,9 +76,9 @@ WORKDIR /hpcc-dev/build
 ARG SIGN_MODULES=OFF
 RUN --mount=type=secret,id=key,uid=10000 if [[ "${SIGN_MODULES}" != "OFF" ]] ; \
     then \
-        gpg --batch --import /run/secrets/key && gpg --list-secret-keys ; \
+    gpg --batch --import /run/secrets/key && gpg --list-secret-keys ; \
```
Comment (GordonSmith) [inline dockerfiles/platform-build/Dockerfile:79]: Whitespace change

File: dockerfiles/platform-build/Dockerfile [hunk 2]
```diff
@@ -76,9 +76,9 @@ WORKDIR /hpcc-dev/build
...
     else \
-        echo "skipping ECL signing" ; \
+    echo "skipping ECL signing" ; \
```
Comment (GordonSmith) [inline dockerfiles/platform-build/Dockerfile:81]: Whitespace change

File: plugins/py3embed/CMakeLists.txt [hunk 3]
```diff
@@ -61,7 +61,7 @@ if (CREATE_STDLIB_ZIP)
     set(_py3_zip_temp_dir "${CMAKE_CURRENT_BINARY_DIR}/python_embeddable_zip_temp")
     set(_py3_zip_path "${_py3_zip_temp_dir}/python${Python3_VERSION_MAJOR}${Python3_VERSION_MINOR}$<$<CONFIG:Debug>:_d>.zip")
 
-    if (NOT "310" STREQUAL "${Python3_VERSION_MAJOR}${Python3_VERSION_MINOR}" )
+    if (NOT "311" STREQUAL "${Python3_VERSION_MAJOR}${Python3_VERSION_MINOR}" )
```
Comment (GordonSmith) [inline plugins/py3embed/CMakeLists.txt:64]: Note:  I did confirm that there was no changes needed to the exclusion list

File: vcpkg_overlays/h3/portfile.cmake [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+vcpkg_cmake_install()
+
+vcpkg_cmake_config_fixup(CONFIG_PATH lib/cmake/${PORT})
+
+vcpkg_copy_pdbs()
+
+file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/bin" "${CURRENT_PACKAGES_DIR}/debug/bin" "${CURRENT_PACKAGES_DIR}/debug/include")
+
+file(INSTALL "${SOURCE_PATH}/LICENSE" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}" RENAME copyright)
```
Comment (GordonSmith) [inline vcpkg_overlays/h3/portfile.cmake:27]: Note:  This file will be removed prior to 9.4.0 (See:  https://github.com/hpcc-systems/HPCC-Platform/pull/17374)

File: vcpkg_overlays/h3/vcpkg.json [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+    {
+      "name": "vcpkg-cmake",
+      "host": true
+    },
+    {
+      "name": "vcpkg-cmake-config",
+      "host": true
+    }
+  ]
+}
```
Comment (GordonSmith) [inline vcpkg_overlays/h3/vcpkg.json:18]: Note:  This file will be removed prior to 9.4.0 (See:  https://github.com/hpcc-systems/HPCC-Platform/pull/17374)

**Comment by ghalliday** [comment]
@GordonSmith it seems to be failing the centos 7 build (for a real reason).

**Comment by GordonSmith** [comment]
@ghalliday - that should be resolved now...

**Review by ghalliday — APPROVED** [review]

**Pull request merged**